src/compiler/transformers/transform-utils.ts

```diff
@@ -776,7 +776,7 @@ export const isStaticGetter = (member: ts.ClassElement): boolean => {
  * @param symbol the `Symbol` to serialize
  * @returns the serialized `Symbol` data
  */
-export const serializeSymbol = (checker: ts.TypeChecker, symbol: ts.Symbol): d.CompilerJsDoc => {
+export const serializeSymbol = (checker: ts.TypeChecker, symbol: ts.Symbol | undefined): d.CompilerJsDoc => {
   if (!checker || !symbol) {
```

Comment:
I'd like to prove out that this check is still needed - gonna keep this in draft form until then (or if I don't have time, we can always add a TODO, although I don't wanna start accumulating a bunch of 'TODO: Update this once SNCs are gone'